examples/extra_features_from_featurizer.ipynb

```diff
@@ -1,5 +1,12 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Extra Features from Featurizer"
```

Comment:
```suggestion
    "# Extra Datapoint Descriptors from Molecule Featurizers",
    "Datapoints can have extra descriptors concatenated to the learned representation before sending to the FFN. These descriptors can be automatically generated using molecule featurizers."
```
In #635 we decided that extra descriptors was less confusing internally in the code than extra features based on how the vector is used. Unfortunately that makes using molecular featurizers a bit more complicated to explain because we use the molecule features as extra descriptors.